cnxeasybake/oven.py

```diff
@@ -1239,7 +1237,7 @@ def split(li, *splitters):
     return [subl for subl in _itersplit(li, splitters) if subl]
 
 
-def css_to_func(css_namespaces, css, flags=None):
+def css_to_func(css, flags, css_namespaces):
```

Comment:
yup, I didn't put `css_namespaces` at the end because I didn't know how to keep the `flags = None`